gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemTest.java

```diff
@@ -54,6 +56,20 @@ public void before() throws IOException {
     postCreateInit();
   }
 
+  @Test
+  public void verifyHadoopPath() throws Exception {
+    GoogleHadoopFileSystem eagerFs =
+        GoogleHadoopFileSystemTestHelper.createInMemoryGoogleHadoopFileSystem();
+    String fileNameWithColon = "empty:file";
+    FSDataOutputStream stream =
+        eagerFs.create(new Path(/* schema= */ null, /* authority= */ null, fileNameWithColon));
+    stream.close();
+    FileStatus[] fileStatus =
+        eagerFs.listStatus(new Path(GoogleHadoopFileSystemTestHelper.IN_MEMORY_TEST_BUCKET));
+    assertThat(fileStatus.length).isEqualTo(1);
```

Comment:
nit: is there `hasLength` assert method?

Reply:
updated.